### PR TITLE
Changed loss function based on regression and classification task 

### PR DESCRIPTION
**Pull request by Anjali295**
Use the train and dev dataset from the data folder for model training and evaluation
Updated the loss function for emotions (BCEWithLogitsLoss due to multi-hot coded values) and personality and IRI scores (MAE and MSE due to continuous values)

### CODE REVIEW

**Review by RBagga1 — APPROVED** [review]
Seems good, thanks Anjali!
